devices.json

```diff
@@ -109,7 +111,8 @@
       "userAgent": "Mozilla/5.0 (iPhone; CPU iPhone OS 8_0 like Mac OS X) AppleWebKit/600.1.3 (KHTML, like Gecko) Version/8.0 Mobile/12A4345d Safari/600.1.4",
       "touch": true,
       "firefoxOS": false,
-      "os": "ios"
+      "os": "ios",
+      "preset": true
```

Comment:
I believe you are supposed to include ""Apple iPhone 6 Plus", not this entry for "Apple iPhone 6s Plus". Thanks Apple for these great product names.